.vscode/tasks.json

```diff
@@ -14,6 +14,47 @@
         "panel": "new"
       },
       "problemMatcher": []
+    },
+    {
+      "label": "Start Addon",
+      "type": "shell",
+      "command": "docker exec -it hassio_cli sh -c \"ha addons stop local_${input:addonName}; ha addons start local_${input:addonName}\"; sudo docker logs --follow addon_local_${input:addonName}",
```

Comment:
```suggestion
      "command": "ha addons stop \"local_${input:addonName}\"; ha addons start \"local_${input:addonName}\"; ha addons logs --follow \"local_${input:addonName}\"",
```

Reply:
@ludeeus Thanks for the suggested improvement! I could not get the last part of the command to work as you suggested, but changed the other parts.

Reply:
What did not work?

Reply:
This part of your suggestion didn't work:

`ha addons logs --follow \"local_${input:addonName}`

It gave me an error, but worked ok when I did it via docker log.

Reply:
What error?
As that syntax is valid.

Reply:
Will take a look after work when I'm not on a mobile device.

Reply:
Hi @ludeeus, here's the settings and the command output for each case:
With:
`"command": "ha addons stop \"local_${input:addonName}\"; ha addons start \"local_${input:addonName}\"; ha addons logs --follow \"local_${input:addonName}\"",
`
I get:
```
 *  Executing task: ha addons stop "local_example"; ha addons start "local_example"; ha addons logs --follow "local_example" 

Command completed successfully.
Command completed successfully.
No systemd-journal-gatewayd Unix socket available
```
--------------------------------
With:
`"command": "ha addons stop \"local_${input:addonName}\"; ha addons start \"local_${input:addonName}\"; docker logs --follow \"addon_local_${input:addonName}\"",`

I get:
```
 *  Executing task: ha addons stop "local_example"; ha addons start "local_example"; docker logs --follow "addon_local_example" 

Command completed successfully.
Command completed successfully.
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
s6-rc: info: service fix-attrs successfully started
s6-rc: info: service legacy-cont-init: starting
s6-rc: info: service legacy-cont-init successfully started
s6-rc: info: service legacy-services: starting
services-up: info: copying legacy longrun example (no readiness notification)
s6-rc: info: service legacy-services successfully started
[23:34:07] INFO: Hello world...
[22:34:07] INFO: Service restart after closing
[23:34:08] INFO: Hello world...
```

Reply:
ah, yes there is no `systemd-journal-gatewayd` there. :(